blog_article/blog_article.adoc

```diff
@@ -0,0 +1,161 @@
+* https://qpid.apache.org/components/jms/index.html[Qpid JMS]:
+Uses Proton-J internally.
+
+As already mentioned, AMQP 1.0 is not based on queues or topics. The target (or source) of a message has to be
+specified using the address field of an AMQP 1.0 message like it is described in the https://github.com/rabbitmq/rabbitmq-server/tree/master/deps/rabbitmq_amqp1_0#routing-and-addressing[plugin documentation].
+
+Since this concept does not exist in AMQP 1.0, the Java clients do not offer the possibility to manage queues,
+topics and bindings like it is possible with the RabbitMQ Java Client for AMQP 0.9.1. Therefore, these entities
+need to already exist when the application is started or can be created using RabbitMQ’s REST API.
+However, the REST API is only available with the help of the management plugin that needs to be enabled.
+
+This approach of managing queues comes along with some constraints in comparison to AMQP 0.9.1 clients: The entities
+that are created with HTTP requests are not client-bound and therefore cannot be created as "exclusive". Automatic
+deletion needs to be achieved through other queue properties like _auto-delete_ and _TTL_. In addition, the REST API does
+not offer to create queues without names to let the broker choose a (temporary) unique queue name. This leads to the
+consequence that unique names have to be managed by the application.  +
+In most cases, these requirements will only be
+needed in specific scenarios. Nevertheless, it should be clear that RabbitMQ in conjunction with AMQP 1.0 can only be
+used with partially limited functionality.
+
+== AMQP 0.9.1 and AMQP 1.0 Interoperability
+As we already know, AMQP 0.9.1 and AMQP 1.0 should be considered as completely different specifications which leads
+to the questions: Is interoperability between these two protocols possible
+"out-of-the-box"? And if not, what has to be considered to achieve interoperability? What about interoperability
+between different clients speaking the same protocol? +
+These points are especially important when we think again about an (evolving) event-driven architecture. The components
+of a system should be able to exchange events and data independently of the protocol they are using. +
+As also described within RabbitMQ’s plugin documentation, interoperability between AMQP 0.9.1 and AMQP 1.0 is provided
+with some constraints:
+
+|===
+|Producer Library |Consumer Library|Interoperability
+
+|SwiftMQ 1.0
+|RabbitMQ 0.9.1
+|provided with constraints -> sending as plain bytes
+
+|RabbitMQ 0.9.1
+|SwiftMQ 1.0
+|provided with constraints -> reading from data field
+
+|Qpid JMS 1.0
+|SwiftMQ 1.0
+|provided
+
+|SwiftMQ 1.0
+|Qpid JMS 1.0
+|provided
+
+|Qpid JMS 1.0
+|RabbitMQ 0.9.1
+|provided with constraints -> sending as plain bytes
+
+|RabbitMQ 0.9.1
+|Qpid JMS 1.0
+|provided
+|===
+
+Like denoted in the table above, when testing the different clients, it was generally possible to send and receive
```

Comment:
I think that it could be helpful to provide some code snippets which show how data needs to be sent and read.